### PR TITLE
Timezone Plugin

### PR DESCRIPTION
**Pull request by zazzaz**

### CODE REVIEW

**Comment by iamkun** [comment]
Many thanks

File: test/plugin/timezone.test.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    const newYork = dayjs.tz('2014-06-01 12:00', NY)
+    expect(newYork.tz('America/Los_Angeles').format()).toBe('2014-06-01T09:00:00-07:00')
+    expect(newYork.tz('Europe/London').format()).toBe('2014-06-01T17:00:00+01:00')
+  })
+})
+
+describe('Convert', () => {
+  it('convert to target time', () => {
+    const losAngeles = dayjs('2014-06-01T12:00:00Z').tz('America/Los_Angeles')
+    const MlosAngeles = dayjs('2014-06-01T12:00:00Z').tz('America/Los_Angeles')
```
Comment (GerardSmit) [inline test/plugin/timezone.test.js:49]: There seems to be a typo here, shouldn't this use `moment.tz` instead of `dayjs`?

Comment (iamkun) [reply test/plugin/timezone.test.js:49]: yep, ha

**Pull request merged**